jobs/gorouter/spec

```diff
@@ -343,7 +343,11 @@ properties:
       are idle for longer than this duration will be closed. Related properties: `router.max_idle_connections`
       and `router.keep_alive_probe_interval`.
     default: 900
-
+  endpoint_dial_timeout_in_seconds:
+    description: |
+      Maximum time in seconds for gorouter to establish a TCP connection with a backend. This timeout comes before `tls_handshake_timeout_in_seconds`
```

Comment:
The setting is also used to wait for read response from WebSockets: https://github.com/cloudfoundry/gorouter/blob/65207dc5dafa659d343975216701fb2e29e3992a/proxy/handler/request_handler.go#L71 -> https://github.com/cloudfoundry/gorouter/blob/main/proxy/handler/forwarder.go#L40 -> https://github.com/cloudfoundry/gorouter/blob/main/proxy/utils/response_reader.go#L30

Should we also document that?

Reply:
There's debate around changing that and making a new timeout specifically for websockets. I am planning on a another PR for that once we assessed the implications of such a change. Meanwhile we can add a statement that this also affects the websocket read timeout and then later remove it again..?

Reply:
@stefanlay I've added a statement for ws timeouts. We can remove it again in a follow-up PR.